torch/_prims/__init__.py

```diff
@@ -1816,6 +1818,69 @@ def _item_meta(a: TensorLikeType) -> TensorMeta:
     doc=_item_doc,
 )
 
+# NOTE: need to model meta scalars
+# See https://github.com/pytorch/pytorch/issues/78070
+def _maximum_value_meta(dtype: torch.dtype) -> TensorMeta:
```

Comment:
This is interesting -- should we model this as a prim or just call torch.finfo?

Reply:
We already have analogous functions in nvfuser. 
https://github.com/csarofeen/pytorch/blob/devel/torch/csrc/jit/codegen/cuda/arith.cpp#L228-L262

It is trickier to map the torch.finfo object in a fusion.